content/4_Gold/MST.mdx

```diff
@@ -144,64 +144,61 @@ no edges are counted twice, we cannot "accidentally" count $n-1$ edges.
 <CPPSection>
 
 ```cpp
-#include <iostream>
-#include <vector>
-#include <algorithm>
+#include <bits/stdc++.h>
 using namespace std;
```

Comment:
can you revert this?

Reply:
tbh i dont think it's more clear to have 5 lines of include statements, but sure